modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -25,6 +25,9 @@ The `domain` value must be unique among all Ingress controllers and cannot be up
 
 If empty, the default value is `ingress.config.openshift.io/cluster` `.spec.domain`.
 
+|`appsDomain`
+|`appsDomain` is an optional domain for AWS infrastructure to use instead of the one specified in the `domain` field when a Route is created without specifying an explicit host. If a value is entered into the `appsDomain`, this value is used to generate default host values for Route. Unlike `domain`, `appsDomain` may be modified after installation. This assumes a new Ingress Controller has been set up with a wildcard certificate.
```

Comment:
```suggestion
|`appsDomain` is an optional domain for AWS infrastructure to use instead of the one specified in the `domain` field when a Route is created without specifying an explicit host. If a value is entered for `appsDomain`, this value is used to generate default host values for the Route. Unlike `domain`, `appsDomain` can be modified after installation. You can use this parameter only if you set up a new Ingress Controller that uses a wildcard certificate.
```
I'm not 100% sure what "This assumes," so please use an accurate clarification if that's not right!

Reply:
That makes sense, thank you!